mod/wikirate/set/type_plus_right/campaign/participant.rb

```diff
@@ -0,0 +1,24 @@
+
+
+# this is pretending as a pointer card
+def item_names _args
+  initiative_name = cardname.left
+  # find all participant and add to the pointer card
+  editors = Card.search editor_of: { left: initiative_name }, return: 'name'
+  editors += Card.search(
+    editor_of: { left: { found_by: "#{initiative_name}+project" } },
+    return: 'name'
+  )
+  editors += Card.search(
+    editor_of: { right_plus:
+      [
+        'Project',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.